examples/general/stats.py

```
@@ -0,0 +1,67 @@
+import mailtrap as mt
+from mailtrap.models.stats import SendingStatGroup
+from mailtrap.models.stats import SendingStats
+from mailtrap.models.stats import StatsFilterParams
+
+API_TOKEN = "YOUR_API_TOKEN"
```

Ruff (0.15.4)

[error] Possible hardcoded password assigned to: "API_TOKEN"

(S105)